frontend/dashboard/src/main.tsx

```diff
@@ -47,16 +47,25 @@ const router = createBrowserRouter([
   },
 ]);
 
+async function enableMocking() {
```

Comment:
Suggest function name chance. Name implies that it always enables mocks. Reader might think this is a bug if they see it in a production context without reading the content of the function. Suggest: startMockingIfConfigured

Reply:
Is it a problem in typescript that one branch returns a promise and the else branch returns undefined?

Reply:
The function isn't awaited and the result is thrown away with the `void` so it's fine here.

`void enableMocking().then(() => {`